### PR TITLE
feat: Add proxmox-ve

### PR DESCRIPTION
**Pull request by jackwasey**
# Description

Add proxmox-ve

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Packaging (updates the packaging)
- [ ] Documentation (updates the documentation)

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have tested my code in common scenarios and confirmed there are no regressions
- [ ] I have added comments to my code, particularly in hard-to-understand sections
- [ ] I have made corresponding changes to the documentation (*remove if no documentation changes were required*)

### CODE REVIEW

File: quickget [hunk 1]
```diff
@@ -2321,6 +2331,17 @@ function get_primtux() {
     echo "${URL}/${ISO} ${HASH}"
 }
 
+function get_proxmox-ve() {
+    local HASH="c96ad84eacbbcef299ab8f407f9602f832abb5ceb08a9aa288c1e1164df2da97"
```
Comment (lj3954) [inline quickget:2335]:
```suggestion
    local HASH=""
```

File: quickget [hunk 2]
```diff
@@ -934,6 +936,10 @@ function editions_primtux() {
     echo 2022-10
 }
 
+function releases_proxmox-ve() {
+    echo 8.2-2
```
Comment (lj3954) [inline quickget:940]:
```suggestion
    echo $(web_pipe https://enterprise.proxmox.com/iso/ | grep proxmox-ve | grep -E -o '[0-9]+\.[0-9]+-[0-9]\.iso' | uniq | sort -ru | cut -d'.' -f 1-2)
```

**Review by lj3954 — CHANGES_REQUESTED** [review]
Dynamically detecting releases is preferred in order to reduce maintenance burden.

**Review by lj3954 — APPROVED** [review]

**Comment by flexiondotorg** [comment]
What is the use case for running Proxmox in a VM? I am cautious about merging this and getting issue reports about VM performance in Proxmox VE under Quickemu.

**Comment by philclifford** [comment]
> What is the use case for running Proxmox in a VM? 

To play with/learn - I mean obviously no-one would do that in production or expect to not have issues of performance or with nesting... 
image

To enable Proxmox-VE users to have a comfortable place to try out upgrades or config changes safely.
To experiment with Proxmox clustering and VM live migration
To enable a further layer of inception possibilities for anyone needing a challenge.
To allow Proxmox users the possibility to contribute to and use `quicktest` on the next version.

**Pull request merged**